README.md

```diff
@@ -194,13 +194,13 @@ const psource = new cql.PatientSource([ {
 } ]);
 
 executor
-  .exec(psource)
-  .then(result => {
-    console.log(JSON.stringify(result, undefined, 2));
-  })
-  .catch(err => {
-    console.error(err);
-  });
+        .exec(psource)
+        .then(result => {
+          console.log(JSON.stringify(result, undefined, 2));
+        })
+        .catch(err => {
+          console.error(err);
+        });
```

Comment:
Indenting 8 spaces looks a bit odd to my eye. I don't think we follow that convention in our code for this project. Can you please revert to the original two-space indenting here? That will also be more consistent with the similar block of code around line 140 above.

Reply:
I have made the changes requested in the second commit.